docker.d/scriptworker.yml

```diff
@@ -27,4 +27,5 @@ task_log_dir: { "$eval": "TASK_LOGS_DIR" }
 watch_log_file: true
 scriptworker_provisioners:
     - { "$eval": "PROVISIONER_ID" }
+    - scriptworker-prov-v1
```

Comment:
Can we drop this stanza instead?

Reply:
Sure, if we don't override `scriptworker_provisioners`, we'll get the right config.

Reply:
oh, you mean to get rid of this section completely?

Reply:
sorry, didn't notice the comment among all 9 PRs

Reply:
Getting rid of this block will work, and may be what we want going forward (if we don't think it's useful to override the defaults). I don't think we have to do that now; it can be a followup if we decide to go this route.